perceval/rendering/pdisplay.py

```diff
@@ -415,6 +415,8 @@ def pdisplay(o, output_format: Format = None, **opts):
                             level is drawn, others are "black boxes"
         - max_v (int): Maximum number of displayed values in distributions
         - sort (bool): if True, sorts a distribution (descending order) before displaying
+        - render_size: In SVG circuit/processor rendering, acts as a zoom factor (float)
```

Comment:
This is the most beautiful documentation I've ever seen. Thanks for this piece of heaven